core/src/main/java/org/openapitools/openapidiff/core/model/ChangedOpenApi.java

```diff
@@ -8,8 +9,8 @@
 import org.openapitools.openapidiff.core.utils.EndpointUtils;
 
 public class ChangedOpenApi implements ComposedChanged {
-  private OpenAPI oldSpecOpenApi;
-  private OpenAPI newSpecOpenApi;
+  @JsonIgnore private OpenAPI oldSpecOpenApi;
+  @JsonIgnore private OpenAPI newSpecOpenApi;
```

Comment:
This would change the output of the JSON renderer and remove the copy of the old and new OpenAPI specifications from the document.

Was this intentional?

Reply:
Yes it was. IMO that information is redundant as they are the inputs to the tool. Plus in the case of large files it makes the output unnecessarily large.